Pod/Classes/ViewControllers/ARArtistComponentViewController.m

```diff
@@ -1,18 +1,29 @@
 #import "ARArtistComponentViewController.h"
+#import "ARGraphQLQueryPreloader.h"
 
 @implementation ARArtistComponentViewController
 
++ (NSDictionary *)propsWithArtistID:(NSString *)artistID;
+{
+    BOOL isPad = UI_USER_INTERFACE_IDIOM() == UIUserInterfaceIdiomPad;
+    return @{ @"artistID": artistID, @"isPad": @(isPad) };
+}
+
++ (NSArray<ARGraphQLQuery *> *)preloadQueriesWithArtistID:(NSString *)artistID;
+{
+    return @[[[ARGraphQLQuery alloc] initWithQueryName:@"QueryRenderersArtistQuery" variables:[self propsWithArtistID:artistID]]];
+}
```

Comment:
The idea is that VCs define a class method that returns the queries and their variables that should be used to preload those queries.